### PR TITLE
Add default value for configuration

### PR DESCRIPTION
**Pull request by ilake**
Add default value for configuration
- Remove the validation we don't need
- Remove the js for prompt/value disable behavior

### CODE REVIEW

**Comment by tubaxenor** [comment]
Can you also rename `Value` to `Default Value` when prompt checkbox checked?

**Comment by ilake** [comment]
Done

File: app/views/project_configurations/_form.html.erb [hunk 1]
```diff
@@ -31,9 +32,9 @@
       Event.observe('configuration_prompt_on_deploy', 'change', function(){
 
         if( $F('configuration_prompt_on_deploy') == '1' ){
-          $('configuration_value').disable();
+          $('config-value').update('Default Value');
```
Comment (tubaxenor) [inline app/views/project_configurations/_form.html.erb:35]: No need for id selector `$('#config-value')` ?

Comment (tubaxenor) [inline app/views/project_configurations/_form.html.erb:35]: And `.update()` works? I thought in jQuery it uses `.text()` or `.html()`

Comment (ilake) [inline app/views/project_configurations/_form.html.erb:35]: Ya, but it is Prototype not jQuery :)

Comment (tubaxenor) [inline app/views/project_configurations/_form.html.erb:35]: Ah I see, :+1: then
2015/1/27 上午11:54 於 "ilake" notifications@github.com 寫道：

**Pull request merged**